pyspectral/atm_correction_ir.py

```diff
@@ -59,6 +59,14 @@
 
 
 import numpy as np
+try:
+    import dask.array as da
+    HAVE_DASK = True
+except ImportError:
+    da = None
+    HAVE_DASK = False
```

Comment:
I just realized, you don't need the `HAVE_DASK` as you could just check `da is None`, **BUT** you don't actually need that check later on anyway. If the data has a `compute` method then we know `da` must not be None because dask must have been used to create the dask arrays.

Reply:
Agree

Reply:
So, @ch-k just remove `HAVE_DASK` as suggested by @djhoese

Reply:
done